api/app/Exceptions/User/UserInvalidPasswordException.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Exceptions\User;
+
+use App\Exceptions\BaseException;
+use Throwable;
+
+final class UserInvalidPasswordException extends BaseException
+{
+    public function __construct($message = "Invalid old password has been provided", $code = 422, Throwable $previous = null)
+    {
+        parent::__construct($message, $code, $previous);
+    }
+}
```

Comment:
Please check the `ExampleException` for the example.

Reply:
Should I do it in this way?
protected $message = 'Invalid old password has been provided';
protected $code = ErrorCode::INVALID_PASSWORD_EXCEPTION;

Reply:
Correct